apps/etterlatte-saksbehandling-ui/client/src/components/oppgavebenk/tildeling/VelgSaksbehandler.tsx

```diff
@@ -77,6 +126,150 @@ export const VelgSaksbehandler = ({ saksbehandlereIEnhet, oppdaterTildeling, opp
+                                onChange={(e) => {
+                                  if (e.target.checked) {
+                                    setValgteOppgaver([...valgteOppgaver, oppgave])
+                                  } else {
+                                    setValgteOppgaver([...valgteOppgaver.filter((o) => o.id !== oppgave.id)])
+                                  }
+                                }}
+                                hideLabel
+                              >
+                                Velg oppgave
+                              </Checkbox>
+                            </Table.DataCell>
+                            <Table.DataCell>{oppgave.sakId}</Table.DataCell>
+                            <Table.DataCell>{formaterDato(oppgave.opprettet)}</Table.DataCell>
+                            <Table.DataCell>{formaterDato(oppgave.frist)}</Table.DataCell>
+                            <Table.DataCell>
+                              {oppgave.fnr ? <PersonLink fnr={oppgave.fnr}>{oppgave.fnr}</PersonLink> : 'Mangler'}
+                            </Table.DataCell>
+                            <Table.DataCell>
+                              <SakTypeTag sakType={oppgave.sakType} kort />
+                            </Table.DataCell>
+                            <Table.DataCell>
+                              <OppgavetypeTag oppgavetype={oppgave.type} />
+                            </Table.DataCell>
+                            <Table.DataCell>
+                              {oppgave.saksbehandler ? oppgave.saksbehandler.navn : 'Ikke tildelt'}
+                            </Table.DataCell>
+                          </Table.Row>
+                        ))}
+                      </Table.Body>
+                    </Table>
+
+                    <HStack gap="4" justify="end">
+                      <Button
+                        variant="secondary"
+                        onClick={lukkGrupperteOppgaver}
+                        disabled={isPending(tildelBulkResult)}
+                      >
+                        Avbryt
+                      </Button>
+
+                      {oppgaver.length === valgteOppgaver.length ? (
+                        <Button
+                          variant="primary"
+                          onClick={() => {
+                            trackClickJaNei(ClickEvent.TILDEL_ALLE_TILKNYTTEDE_OPPGAVER, JaNei.JA)
+                            tildel(valgtSaksbehandler!!)
+                          }}
+                          loading={isPending(tildelBulkResult)}
+                        >
+                          Ja, tildel alle
+                        </Button>
+                      ) : (
+                        <Button
+                          variant="primary"
+                          onClick={() => {
+                            trackClickJaNei(ClickEvent.TILDEL_ALLE_TILKNYTTEDE_OPPGAVER, JaNei.NEI)
```

Comment:
Skulle man hatt mer forklarende svar enn Ja og Nei på målingene her? 
Er ikke nødvendigvis så lett å se at Nei betyr Tildel valgte når man se i Amplitude. 

Vet ikke om det er mulig å gjenbruke mer av koden her, for nå er det ish 3 ord forskjell på 20 linjer med kode

Reply:
Slo meg plutselig at vi også burde ta høyde for at tildeling avbrytes. Så da gjorde jeg om på svaret som logges allikevel. 

Gjenbrukte også litt kode for opprydding 👍